static/js/https/domains.js

```diff
@@ -79,118 +83,168 @@ $(document).ready(function () {
+
+    else
+      details = "";
 
-    return details;
+    // If there's an F grade, and TLS details weren't already included,
+    // add an urgent warning.
+    if (urgent)
+      return details + " " + w("Warning: ") + l(labsUrlFor(row.canonical), "review SSL Labs report") + " to resolve TLS quality issues."
+    else
+      return details;
   };
 
   var links = {
-    rc4: "https://https.cio.gov/technical-guidelines/#rc4",
     hsts: "https://https.cio.gov/hsts/",
     sha1: "https://https.cio.gov/technical-guidelines/#signature-algorithms",
     ssl3: "https://https.cio.gov/technical-guidelines/#ssl-and-tls",
-    tls: "https://https.cio.gov/technical-guidelines/#ssl-and-tls",
-    fs: "https://https.cio.gov/technical-guidelines/#forward-secrecy",
-    preload: "https://https.cio.gov/hsts/#hsts-preloading"
+    tls12: "https://https.cio.gov/technical-guidelines/#ssl-and-tls",
+    preload: "https://https.cio.gov/hsts/#hsts-preloading",
+    stay_preloaded: "https://hstspreload.appspot.com/#continued-requirements",
+    submit: "https://hstspreload.appspot.com"
   };
 
   var l = function(slug, text) {
     return "<a href=\"" + (links[slug] || slug) + "\" target=\"blank\">" + text + "</a>";
   };
 
-  // Mention a few high-impact TLS issues that will have affected
-  // the SSL Labs grade.
-  var tlsDetails = function(data, type, row) {
-    if (type == "sort")
-      return null;
-
-    if (row.https.grade < 0)
-      return "No data.";
-
-    var config = [];
-
-    if (row.https.uses == 1)
-      config.push("uses a certificate chain that may be invalid for some visitors");
-
-    if (row.https.sig == "SHA1withRSA")
-      config.push("uses a certificate with a " + l("sha1", "weak SHA-1 signature"));
-
-    if (row.https.ssl3 == true)
-      config.push("supports the " + l("ssl3", "insecure SSLv3 protocol"));
-
-    if (row.https.rc4 == true)
-      config.push("supports the " + l("rc4", "deprecated RC4 cipher"));
-
-    if (row.https.tls12 == false)
-      config.push("lacks support for the " + l("tls", "most recent version of TLS"));
-
-    // Don't bother remarking if FS is Modern or Robust.
-    if (row.https.fs <= 1)
-      config.push("should enable " + l("fs", "forward secrecy"));
-
-    var issues = "";
-    if (config.length > 0)
-      issues += "This domain " + config.join(", ") + ". ";
-
-    issues += "See the " + l(labsUrlFor(row.domain), "full SSL Labs report") + " for details.";
+  var g = function(text) {
+    return "<strong class=\"great\">" + text + "</strong>";
+  };
 
-    return issues;
+  var w = function(text) {
+    return "<strong class=\"warning\">" + text + "</strong>";
   };
 
-  var detailsKeyboardCtrl = function(){
-      $('table tbody tr td:first-child').attr('tabindex','0')
-      .attr('aria-label','Select for additional details')
-      .on('keydown',function(e){
-        if (e.keyCode == 13)
-          $(this).click();
-          $(this).parent().next('tr.child').focus();
-      })
-    };
```

Comment:
I really like this cleanup so we don't have to have this show/hide/keyboard click hack anymore.